src/org-manager/containers/pending-organisations/pending-organisations.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Store, select } from '@ngrx/store';
+import { GovukTableColumnConfig } from 'projects/gov-ui/src/lib/components/govuk-table/govuk-table.component';
+import { Observable, Subscription } from 'rxjs';
+import 'rxjs/add/observable/of';
+import * as fromStore from '../../../org-manager/store';
+import * as fromRoot from '../../../app/store';
+import { PendingOverviewColumnConfig } from 'src/org-manager/config/pending-overview.config';
+import { OrganisationVM } from 'src/org-manager/models/organisation';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import * as fromOrganisation from '../../store/';
+import {takeWhile} from 'rxjs/operators';
+@Component({
+  selector: 'app-pending-overview-component',
+  templateUrl: './pending-organisations.component.html',
+})
+
+export class PendingOrganisationsComponent implements OnInit {
+  columnConfig: GovukTableColumnConfig[];
+  pendingOrgs$: Observable<OrganisationVM[]>;
+  inputForm: FormGroup;
+  errorMessage$: Observable<string>;
+  loading$: Observable<boolean>;
+
+  constructor(public store: Store<fromStore.OrganisationRootState>,
```

Comment:
shouldn't store be private?

Reply:
nope because it gets passed on to child component to executes actions so needs to be accessible... not sure if this is the best way but have left it for now as it was done before... maybe does need more thoughts

Reply:
OK, we'll leave this one for now then.